index.html

```diff
@@ -5,39 +5,39 @@
     <title>Kate Lucas</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link href="https://fonts.googleapis.com/css?family=Arimo" rel="stylesheet" type="text/css">
-    <link href="vendor/normalize.css" type="text/css" rel="stylesheet">
-    <link href="vendor/hover.css" type="text/css" rel="stylesheet">
+    <link href="vendor/styles/normalize.css" type="text/css" rel="stylesheet">
+    <link href="vendor/styles/hover.css" type="text/css" rel="stylesheet">
     <link href="styles/base.css" type="text/css" rel="stylesheet">
     <link href="styles/modules.css" type="text/css" rel="stylesheet">
     <link href="styles/style.css" type="text/css" rel="stylesheet">
   </head>
   <body align=center>
-    <nav class='main-nav'>
-      <img class='hamburger' src='img/ham.png'>
-      <ul class='nav'>
-        <li class='tab' data-content='home'><a href='#home' class='hvr-grow'>home</a><li>
-        <li class='tab' data-content='about'><a href='#about' class='hvr-grow'>about</a><li>
-        <li class='tab' data-content='projects'><a href='#projects' class='hvr-grow'>projects</a><li>
+    <nav class="main-nav">
+      <img class="hamburger" src="img/ham.png">
+      <ul class="nav">
+        <li class="tab" data-content="home"><a href="/home" class="hvr-grow">home</a><li>
```

Comment:
Your home route should just be a /. Not a /home.  That way when someone first navigates to your site they will hit the home route.